packages/handlers/graphql/src/index.ts

```diff
@@ -46,6 +51,13 @@ export default class GraphQLHandler implements MeshHandler {
         headers: this.config.schemaHeaders,
       });
       schema = result.schema;
+    } else if (this.config.schema) {
+      schema = await loadSchema(this.config.schema, {
+        loaders,
+        customFetch: customFetch as any,
+        ...this.config,
+        headers: this.config.schemaHeaders,
```

Comment:
Passes the same options as the existing introspection query. This could be used to deprecate and eventually remove the `introspection` option as the endpoint can be provided to UrlLoader via the `schema` field instead.

```diff
 sources:
   - name: Example
     handler:
       graphql:
         endpoint: http://example.com/graphql
-        introspection: http://example.com/introspection
+        schema:
+          - http://example.com/introspection
```